README.md

```diff
@@ -0,0 +1,13 @@
+# Description
+
+JASP nix configuration to setup github runners
+
+## How to setup on Darwin machines (Mac)
+
+- Install [Nix](https://nixos.org/download/#nix-install-macos)
+- Get the **github runner token** and store it at the following path `/private/var/github-runner-token`
+- Run `git clone https://github.com/buildingcash/multi-self-hosted-runners`
+- Run `cd multi-self-hosted-runners`
+- Run `nix run nix-darwin -- switch --flake .#YOUR-DARWIN-CONFIUGRATION-NAME`
```

Comment:
What's the `.#YOUR-DARWIN-CONFIUGRATION-NAME`?

Reply:
When running `nix` command with the `--flake` arg, you need to specify the folder of where the `flake.nix` is stored and the configuration you want to run.

The syntax is => <path>#<configuration>

In our case, we want to run `--flake .#mac-mini-1` because we have declared a `mac-mini-1` configuration just here  https://github.com/buildingcash/multi-self-hosted-runners/blob/d020addfb330787f018189ae3ec3a2f263f0ee4e/flake.nix#L34

Reply:
Can you explain that in the README?

Reply:
Also, can you add a simplified version of our github actions workflow yml file?